app/controllers/observations_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class ObservationsController < ApplicationController
+  # These need to be moved into the files where they are actually used.
+  require "find"
+  require "set"
+
+  include Index
+  include Show
+  include NewAndCreate
+  include EditAndUpdate
+  include Destroy
+  include Suggestions
+  include Map
+  include Download
+
+  # Disable cop: all these methods are defined in files included above.
+  # rubocop:disable Rails/LexicallyScopedActionFilter
+
+  before_action :login_required, except: [
+    :show
+  ]
+
+  before_action :disable_link_prefetching, except: [
+    :create,
```

Comment:
I think you mean `:new` not `:create`.  Isn't new the one that serves the form and create the one that actually creates the object?  (And edit serves the form while update modifies the object?)